components/multiselect/multiselect.tsx

```diff
@@ -495,10 +506,24 @@ export default class MultiSelect<T extends Value> extends React.PureComponent<Pr
                 </div>
                 {this.props.saveButtonPosition === 'bottom' &&
                 <div className='multi-select__footer'>
+                    {
+                        typeof this.props.backButtonClick === 'function' &&
```

Comment:
Is there any reason why we can't use just `this.props.backButtonClick`, I would expect it to be a function or `undefined` (and that is what the type check expects too).

Reply:
Fixed, not sure why I was using `typeof this.props.backButtonClick === 'function'`